text/src/autogluon/text/text_prediction/presets.py

```diff
@@ -12,18 +12,24 @@ def list_text_presets(verbose=False):
         "default": {
             "model.hf_text.checkpoint_name": "google/electra-base-discriminator",
             "optimization.lr_decay": 0.9,
+            "model.hf_text.max_text_len": 256,
```

Comment:
Does the change (512 -> 256) affect performance? 512 works well on our benchmark datasets.